backend_addon/{{ cookiecutter.__folder_name }}/pyproject.toml

```diff
@@ -1,15 +1,98 @@
-# Generated from:
-# https://github.com/plone/meta/tree/main/config/default
-# See the inline comments on how to expand/tweak this configuration file
+[project]
+name = "{{ cookiecutter.python_package_name }}"
+dynamic = ["version"]
+description = "{{ cookiecutter.description }}"
+readme = "README.md"
+license = "GPL-2.0-only"
+requires-python = ">={{ cookiecutter.__supported_versions_python[0] }}"
+authors = [
+    { name = "{{ cookiecutter.author }}", email = "{{ cookiecutter.email }}" },
+]
+keywords = [
+    "CMS",
+    "Plone",
+    "Python",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Web Environment",
+    "Framework :: Plone",
+    {%- for version in cookiecutter.__supported_versions_plone -%}
+    "Framework :: Plone :: {{ version }}",
```

Comment:
Broke for me with:
```
ValueError: Unknown classifier in field `project.classifiers`: Framework :: Plone :: 6.1.0
```

Reply:
Fixed by adding a new filter to cookieplone 0.9.3 and use it here